[Bug]: Test case test_subvolume_group_pinning failing with KeyError: 'distributed'

### Description
The test case tests/functional/storageclass/test_csi_subvolume_group_property.py::TestCSISubvolumeGroup::test_subvolume_group_pinning is failing with error given below

```
def test_subvolume_group_pinning(self):
    """
    Test that verifies that the pinning value of CephFilesystemSubVolumeGroup is 1
    """

    result = run_oc_command("get CephFilesystemSubVolumeGroup -o yaml")
    yaml_dict = yaml.safe_load("\n".join(result))
    try:

      pinning_val = yaml_dict["items"][0]["spec"]["pinning"]["distributed"]
E           KeyError: 'distributed'



tests/functional/storageclass/test_csi_subvolume_group_property.py:23: KeyError


During handling of the above exception, another exception occurred:


self = <tests.functional.storageclass.test_csi_subvolume_group_property.TestCSISubvolumeGroup object at 0x7f16e92dca30>


def test_subvolume_group_pinning(self):
    """
    Test that verifies that the pinning value of CephFilesystemSubVolumeGroup is 1
    """

    result = run_oc_command("get CephFilesystemSubVolumeGroup -o yaml")
    yaml_dict = yaml.safe_load("\n".join(result))
    try:
        pinning_val = yaml_dict["items"][0]["spec"]["pinning"]["distributed"]
    except KeyError as e:
        err_msg = 'Pinning property not found, missing key "%s"' % str(e)
        logger.error(err_msg)

      raise Exception(err_msg)
E           Exception: Pinning property not found, missing key "'distributed'"



tests/functional/storageclass/test_csi_subvolume_group_property.py:27: Exception
```

### Steps to Reproduce
1.Run the test case tests/functional/storageclass/test_csi_subvolume_group_property.py::TestCSISubvolumeGroup::test_subvolume_group_pinning

### Actual Behavior
Test case is failing because the spec the test case is looking for is not present.

```
% oc get cephfilesystemsubvolumegroup csi -o=jsonpath="{.spec}" | jq
{
  "dataPoolName": "",
  "filesystemName": "ocs-storagecluster-cephfilesystem",
  "name": "csi",
  "pinning": {}
}

```
### Expected Behavior

The parameter "pinning": "distributed=1" is present in the status. If this is a new change, the test case should be updated. 
```
% oc get cephfilesystemsubvolumegroup csi -o=jsonpath="{.status.info}" | jq
{
  "clusterID": "5bb69c306a7d011c3e91c3cec112fb7a",
  "pinning": "distributed=1"
}

```

### Impact (likelihood of reproduction, impact on the cluster and on other tests, etc.)
Not impacting the cluster or other tests
### Screenshots (if applicable)

### Environment
- Test Suite(s): tier3
- Platform(s): Reported in IBM Cloud
- Version(s): ODF and OCP 4.19.0
- OS:

### Additional Context
Report portal https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/852/32279/1586096/1586170/log

--- a/tests/functional/storageclass/test_csi_subvolume_group_property.py
+++ b/tests/functional/storageclass/test_csi_subvolume_group_property.py
@@ -2,7 +2,7 @@ import logging
 import yaml
 
 from ocs_ci.framework.testlib import ManageTest, tier3, skipif_external_mode
-from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.framework.pytest_customization.marks import green_squad, jira, polarion_id
 from ocs_ci.helpers.performance_lib import run_oc_command
 
 logger = logging.getLogger(__name__)
@@ -12,6 +12,8 @@ logger = logging.getLogger(__name__)
 @skipif_external_mode
 @tier3
 class TestCSISubvolumeGroup(ManageTest):
+    @jira("DFBUGS-2759")
+    @polarion_id("OCS-5740")
     def test_subvolume_group_pinning(self):
         """
         Test that verifies that the pinning value of CephFilesystemSubVolumeGroup is 1

--- a/tests/functional/storageclass/test_csi_subvolume_group_property.py
+++ b/tests/functional/storageclass/test_csi_subvolume_group_property.py
@@ -1,7 +1,7 @@
 import logging
 import yaml
 
-from ocs_ci.framework.testlib import ManageTest, tier3, skipif_external_mode
+from ocs_ci.framework.testlib import ManageTest, tier2, skipif_external_mode
 from ocs_ci.framework.pytest_customization.marks import green_squad, jira, polarion_id
 from ocs_ci.helpers.performance_lib import run_oc_command
 
@@ -10,7 +10,7 @@ logger = logging.getLogger(__name__)
 
 @green_squad
 @skipif_external_mode
-@tier3
+@tier2
 class TestCSISubvolumeGroup(ManageTest):
     @jira("DFBUGS-2759")
     @polarion_id("OCS-5740")

